data_ocean/fixtures/register.json

```
@@ -8,269 +8,115 @@
       "name_eng": "Register of Politically Exposed Persons",
       "source_name": "pep.org.ua",
       "source_register_id": null,
-      "url_address": "https://pep.org.ua/uk/data",
-      "api_address": "https://pep.org.ua/opendata/persons/json",
-      "source_last_update": "2020-09-07"
-    }
-  },
-  {
-    "model": "data_ocean.dataset",
-    "pk": 1,
-    "fields": {
-      "created_at": "2020-09-01 15:40",
-      "name": "all PEPs list",
-      "endpoint": "/api/pep",
-      "type": "list",
-      "register_id": 1
-    }
-  },
-  {
-    "model": "data_ocean.dataset",
-    "pk": 2,
-    "fields": {
-      "created_at": "2020-09-10 15:40",
-      "name": "PEP",
-      "endpoint": "/api/pep/{id}",
-      "type": "retrieve",
-      "register_id": 1
+      "source_url_address": "https://pep.org.ua/uk/data",
+      "source_api_address": "https://pep.org.ua/opendata/persons/json",
+      "api_list": "/api/pep",
+      "api_detail": "/api/pep/{id}"
```

it is good practice to put a slash at the end of the url
```suggestion
      "api_list": "/api/pep/",
      "api_detail": "/api/pep/{id}/"
```

---

done